core/src/cats/cats.h

```diff
@@ -536,9 +522,7 @@ typedef int(DB_RESULT_HANDLER)(void*, int, char**);
 
 class pathid_cache;
 
-/*
- * Initial size of query hash table and hint for number of pages.
- */
+// Initial size of query hash table and hint for number of pages.
```

Comment:
if you look below at line 529-531 and 535-537 you will notice your script misses at least some comments.